Runtime/Behaviours/CinemachineConfiner.cs

```diff
@@ -62,6 +62,11 @@ public enum Mode
         [Range(0, 10)]
         public float m_Damping = 0;
         
+        /// <summary>
+        /// The amount with which the confiner displaces the vcam.
+        /// </summary>
+        public Vector3 m_Displacement { get; private set; }
```

Comment:
This is incorrect.  It's a per-vcam state, so must live in the VcamExtraState.
Naming: remove the m_
Also the description should be: The amount _by which_ the confiner...